cmd/oceancenter/dl/hydrophone/pkg/v1.0.1/pkg.json

```diff
@@ -0,0 +1,14 @@
+{
+  "device": "hydrophone",
+  "description": "Install rv in /opt/ausocean/bin.",   
+  "version": "v1.0.0",
+  "components": [
+    {
+    "name": "rv",
```

Comment:
Make _rv/v1.6.1_ a shared component.

Reply:
Isn't that what I've done here by adding 
"shared": true,
?